cloud

```diff
@@ -126,6 +126,7 @@ function execute_command() {
     declare -r cache_bucket_name="${project}_${prefix}cache"
     declare -r pick_notifications_trigger_topic="${prefix}pick_notifications_trigger"
     declare -r purge_db_trigger_topic="${prefix}purge_db_trigger"
+    declare -r topic="playground_kcidb_new"
```

Comment:
Now, you don't hardcode the topic name, you use the submission topic of this deployment. Then when we deploy to playground it will be the one you use here.

You simply need to use the `new_topic` defined below, instead.

Reply:
Thanks for the pointers.